packages/node-core/src/indexer/storeCache/cachePoi.ts

```diff
@@ -1,35 +1,35 @@
 // Copyright 2020-2022 OnFinality Limited authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+import {DEFAULT_FETCH_RANGE} from '@subql/common';
 import {u8aToBuffer} from '@subql/utils';
 import {Mutex} from 'async-mutex';
 import {Op, Transaction} from 'sequelize';
 import {getLogger} from '../../logger';
 import {PoiRepo, ProofOfIndex} from '../entities';
+import {PoiInterface} from '../poi/poiModel';
 import {ICachedModelControl} from './types';
-
 const logger = getLogger('PoiCache');
 
-const DEFAULT_FETCH_RANGE = 100;
-
-export class CachePoiModel implements ICachedModelControl {
+export class CachePoiModel implements ICachedModelControl, PoiInterface {
```

Comment:
I think it would be a good idea for this to have a private `PlainPoiModel` then rather than making queries on the model here we can use the queries from `PlainPoiModel`. That way we only define the queries once